src/s2python/s2_connection.py

```diff
@@ -1,3 +1,8 @@
+try:
+    import websockets
+except ImportError as exc:
+    raise ImportError("You need to run 'pip install s2-python[ws]' to use this feature.") from exc
```

Comment:
```suggestion
    raise ImportError("websockets is required. This can be installed by running 'pip install s2-python[ws]'") from exc
```

Reply:
The above is just a suggestion, but I think it would be useful to indicate exatly what's missing.

Reply:
The `from exc` also notes the exact package that is missing, but I agree it's better to be a little redundant just to have it all visible on one log line.